osx/build-on-jenkins.sh

```diff
@@ -15,10 +15,10 @@ pushd $D/src
     mv org.jenkins-ci.plis_ $OSX_IDPREFIX.plist
   popd
 popd
-$BASE/bin/branding.sh $D/src
+"$BASE/bin/branding.py" $D/src
```

Comment:
use branding.py which is much better at not needing to escape the escpaes of escapes to pass to shells at various levels.. URGGG...